test/x3/list.cpp

```diff
@@ -89,10 +89,8 @@ main()
     }
 
     { // actions
-        using boost::spirit::x3::_attr;
-
         std::string s;
-        auto f = [&](auto& ctx){ s = std::string(_attr(ctx).begin(), _attr(ctx).end()); };
+        auto f = [&](auto& ctx){ s = std::string(boost::spirit::x3::_attr(ctx).begin(), boost::spirit::x3::_attr(ctx).end()); };
 
```

Comment:
Because (without the full qualification) it results in internal compiler error. I just wanted to fix as much tests as I could. - I'm ok with rolling this change back as it is just a test.